akka-actor-typed-tests/src/test/scala/akka/actor/typed/scaladsl/RoutersSpec.scala

```diff
@@ -115,6 +115,41 @@ class RoutersSpec extends ScalaTestWithActorTestKit("""
       }
     }
 
+    "supports broadcast" in {
+      trait Cmd
+      case object ReplyWithAck extends Cmd
+      case class BCast(cmd: Cmd) extends Cmd
+
+      val atomicInt = new AtomicInteger(0)
+      val probe = testKit.createTestProbe[AnyRef]()
+
+      val behavior = Behaviors.receiveMessage[Cmd] {
+        case ReplyWithAck =>
+          val reply = Integer.valueOf(atomicInt.incrementAndGet())
+          probe.ref ! reply
+          Behaviors.same
+        case BCast(BCast(_)) =>
+          fail("totally unacceptable")
```

Comment:
This wouldn't be noticed by the test, the routee would crash and nobody would know.